server/routes/razor.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express");
```

Comment:
don't use name as razor, keep it generic
use 'payment' name instead

Reply:
I have changed to it